ui/components/app/assets/asset-list/network-filter/network-filter.tsx

```diff
@@ -125,10 +154,9 @@ const NetworkFilter = ({ handleClose }: SortControlProps) => {
               type="PRIMARY"
               ethNumberOfDecimals={4}
               hideTitle
+              showFiat
               shouldCheckShowNativeToken
-              isAggregatedFiatOverviewBalance={
-                !showNativeTokenAsMainBalance && !isTestnet
-              }
+              isAggregatedFiatOverviewBalance={!isTestnet}
```

Comment:
Noticed some weird behavior here:

<img width="392" alt="Screenshot 2024-11-18 at 5 51 18 PM" src="https://github.com/user-attachments/assets/07169ac9-a8a6-456e-86e8-ae1bfc0583a1">

I think we can get around this by either wrapping the `UserPreferencedCurrencyDisplay` in a Text component, or use the `textProps` to consistently style the text.

```
            <Text
              variant={TextVariant.bodySmMedium}
              color={TextColor.textAlternative}
            >
              <UserPreferencedCurrencyDisplay
                value={selectedAccountBalance}
                type="PRIMARY"
                ethNumberOfDecimals={4}
                hideTitle
                showFiat
                shouldCheckShowNativeToken
                isAggregatedFiatOverviewBalance={!isTestnet}
              />
            </Text>
```